pkg/stages/stages.go

```diff
@@ -732,21 +742,13 @@ depend() {
 
 func GetServicesStage(_ values.System, _ types.KairosLogger) []schema.Stage {
 	return []schema.Stage{
-		{
-			Name:     "Enable services for Modern systems",
-			OnlyIfOs: "Ubuntu.*|Debian.*|Fedora.*",
-			Systemctl: schema.Systemctl{
-				Enable: []string{
```

Comment:
All of this is already covered in other entries, ubuntu+debian under the debian family and Fedora under RHEL family :D